bftengine/src/bcstatetransfer/DBDataStore.cpp

```diff
@@ -401,6 +401,22 @@ void DBDataStore::deleteCoveredResPageInSmallerCheckpoints(uint64_t minChkp) {
   inmem_->deleteCoveredResPageInSmallerCheckpoints(minChkp);
 }
 
+void DBDataStore::clearDataStoreData() {
+  del(Initialized);
```

Comment:
it's enough to delete Initialized only

Reply:
@f-squirrel recommended to delete it all, so if it isn't matter I will keep it that way.
As for the file storage, I will ask @yuliasherman but in any case, this is not part of this PR.